### ISSUE TITLE
Markdown compatibility problem: Hard break

### ISSUE THREAD

**Issue by bmc** [tristanjuricek/knockoff#23]
Take the following input:

```
This is a line ending with two blanks.  
That should produce a hard "br" in Markdown, per Daring Fireball.  
It doesn't.
```

The three lines end with two blank characters. Per Daring Fireball, that's supposed to cause a hard break. Here's what the Perl Markdown script produces, given that input:

```
<p>This is a line ending with two blanks.<br />
That should produce a hard "br" in Markdown, per Daring Fireball.<br />
It doesn't.</p>
```

Here's what Knockoff produces:

```
<p>This is a line ending with two blanks.  
</p><p>That should produce a hard &quot;br&quot; in Markdown, per Daring Fireball.  
</p><p>It doesn't.</p>
```

**Comment by tristanjuricek** [comment]
Gah, I was apparently going through my illiterate phase when I reinterpreted that to mean "make everything a paragraph". Sorry, should be fixed quickly.

**Comment by bmc** [comment]
No worries. CSS can be used as a workaround.

**Comment by tristanjuricek** [comment]
OK, should be fixed in release 0.7.2-13

**Comment by bmc** [comment]
Fix confirmed here. Thanks, Tristan.
